tap_gitlab/schemas/vulnerabilities.json

```diff
@@ -0,0 +1,491 @@
+      "anyOf": [
+        {
+          "type": "string",
+          "format": "date-time"
+        },
+        {
+          "type": "null"
+        }
+      ]
+    },
+    "description": {
+      "type": [
+        "null",
+        "string"
+      ]
+    },
+    "dismissed_at": {
+      "anyOf": [
+        {
+          "type": "string",
+          "format": "date-time"
+        },
+        {
+          "type": "null"
+        }
+      ]
+    },
+    "dismissed_by_id": {
+      "type": [
+        "null",
+        "integer"
+      ]
+    },
+    "due_date": {
+      "anyOf": [
+        {
+          "type": "string",
+          "format": "date-time"
+        },
+        {
+          "type": "null"
+        }
+      ]
+    },
+    "finding": {
+      "type": "object",
+      "properties": {
+        "confidence": {
+          "type": [
+            "null",
+            "string"
+          ]
+        },
+        "created_at": {
+          "anyOf": [
+            {
+              "type": "string",
+              "format": "date-time"
+            },
+            {
+              "type": "null"
+            }
+          ]
+        },
+        "id": {
+          "type": [
+            "null",
+            "integer"
+          ]
+        },
+        "location_fingerprint": {
+          "type": [
+            "null",
+            "string"
+          ]
+        },
+        "metadata_version": {
+          "type": [
+            "null",
+            "string"
+          ]
+        },
+        "name": {
+          "type": [
+            "null",
+            "string"
+          ]
+        },
+        "primary_identifier_id": {
+          "type": [
+            "null",
+            "integer"
+          ]
+        },
+        "project_fingerprint": {
+          "type": [
+            "null",
+            "string"
+          ]
+        },
+        "project_id": {
+          "type": [
+            "null",
+            "integer"
+          ]
+        },
+        "raw_metadata": {
+          "type": [
+            "null",
+            "string"
+          ]
+        },
+        "report_type": {
+          "type": [
+            "null",
+            "string"
+          ]
+        },
+        "scanner_id": {
+          "type": [
+            "null",
+            "integer"
+          ]
+        },
+        "severity": {
+          "type": [
+            "null",
+            "string"
+          ]
+        },
+        "updated_at": {
+          "anyOf": [
+            {
+              "type": "string",
+              "format": "date-time"
+            },
+            {
+              "type": "null"
+            }
+          ]
+        },
+        "uuid": {
+          "type": [
+            "null",
+            "string"
+          ]
+        },
+        "vulnerability_id": {
+          "type": [
+            "null",
+            "integer"
+          ]
+        }
+      }
+    },
+    "author_id": {
+      "type": [
+        "null",
+        "integer"
+      ]
```

Comment:
@benomahony looks like there are some duplicate fields

Reply:
Yup, reviewed against example schema here: https://docs.gitlab.com/ee/api/project_vulnerabilities.html
This will be why it took me so long 🤦

Reply:
Fixed